vllm/v1/engine/core_client.py

```diff
@@ -416,9 +416,9 @@ def __init__(self, vllm_config: VllmConfig, executor_class: type[Executor],
 
         def process_outputs_socket():
             shutdown_socket = ctx.socket(zmq.PAIR)
-            shutdown_socket.bind(shutdown_path)
             out_socket = make_zmq_socket(ctx, output_path, zmq.constants.PULL)
             try:
+                shutdown_socket.bind(shutdown_path)
```

Comment:
QQ: What is this change for?

Reply:
This was just a mistake in the original ordering.. it's best to have inside the try block so that the socket gets closed if the bind call raises (admittedly unlikely).

Reply:
Got it. Thanks for the explanation!